src/pages/en/guides/cms/storyblok.md

```diff
@@ -1,14 +1,486 @@
+
+To render our content, pass the `content` property of the story to the `StoryblokComponent` as `blok`. This component will render the bloks that are defined inside the `content` property. In our case, it will render the `BlogPost` component.
+
+## Making a blog with Astro and Storyblok
+
+With the integration set up, we can now create a blog with Astro and Storyblok.
+
+### Prerequisites
+
+1. A Storyblok space - For this tutorial, we recommend using a new space. If you already have a space with bloks, feel free to use them, but you will need to modify the code to match the blok names and content types.
+
+2. An Astro project integraded with Storyblok - See [integrating with Astro](#integrating-with-astro) for instructions on how to set up the integration.
+
+### Creating a blok library
+
+To create bloks, go to the Storyblok app and click on the **Block Library** tab. Click on <kbd>+ New blok</kbd> button and create the following bloks:
+
+1. `blogPost` - A content type blok with the following fields:
+    - `title` - A text field
+    - `description` - A text field
+    - `content` - A rich text field
+
+2. `blogPostList` - An empty nestable blok
+
+3. `page` - A content type blok with the following fields:
+    - `body` - A nestable blok
+
+### Creating content
+
+Using the blok library that we created in the previous step, create the following content:
+
+1. `home` - A content type story with the `page` blok. Inside the `body` field, add a `blogPostList` blok.
+
+2. `blog` - A folder to store our blog posts in.
+
+3. `blog/no-javascript` - A story with the `blogPost` content type.
+    ```yaml
+    title: No JavaScript
+    description: A sample blog post
+    content: Hi there! This blog post doesn't use JavaScript.
+    ```
+3. `blog/astro-is-amazing` - A story with the `blogPost` content type.
+    ```yaml
+    title: Astro is amazing
+    description: We love Astro
+    content: Hi there! This blog post was build with Astro.
+    ```
+
+Now that we have our content ready, we can switch to our Astro project and start building our blog.
+
+### Connecting bloks to components
+
+To connect your newly created bloks to Astro components, create a new folder named `storyblok` in your `src` directory and add the following files:
+
+
+```astro title="src/storyblok/Page.astro"
+---
+import { storyblokEditable } from '@storyblok/astro'
+import StoryblokComponent from '@storyblok/astro/StoryblokComponent.astro'
+const { blok } = Astro.props
+---
+
+<main {...storyblokEditable(blok)}>
+  {
+    blok.body?.map((blok) => {
+      return <StoryblokComponent blok={blok} />
+    })
+  }
+</main>
+```
+
+`Page.astro` is a nested component that will recursively render all the bloks inside the `body` property of the `page` blok. It also adds the `storyblokEditable` directive to the parent element which will allow us to edit the page in Storyblok.
```

Comment:
nested component refers to a `Blok` inside a `Blok`. they call them nestable

Reply:
Gotcha, I was just wondering if it would be confusing to call the `.astro` component a nested component if that was a Storyblok thing. e.g. `Page.astro` _represents_ a nested component.